src/odin/enhancedtrippath.cc

```diff
@@ -123,6 +123,7 @@ const std::string& TripLeg_TravelMode_Name(int v) {
 const std::string& TripLeg_VehicleType_Name(int v) {
   static const std::unordered_map<int, std::string> values{
       {0, "kCar"}, {1, "kMotorcycle"}, {2, "kAutoBus"}, {3, "kTractorTrailer"}, {4, "kMotorScooter"},
+      {5, "kFourWheelDrive"}, {6, "kGolfCart"},
```

Comment:
I think the 4WD value was always missing, but it was never noticed ;)

Reply:
in fact I removed it recently:) maybe you're not on latest master?

Reply:
That could have also gotten lost in merge conflict resolution 🎃 If it's gone, then I guess I can set this constant to 5 as well?

Reply:
Should be gone now.